app/views/comments/create.js.erb

```diff
@@ -0,0 +1,11 @@
+$(document).ready(function(){
```

Comment:
Since this is being executed using AJAX, we don't need wrap this in a function and wait for `$(document).ready`. It gets executed when we make the remote web request.